generated/nidmm/nidmm.proto

```diff
@@ -497,6 +514,11 @@ enum ThermocoupleType {
   THERMOCOUPLE_TYPE_NIDMM_VAL_TEMP_TC_T = 11;
 }
 
+enum TimeLimit {
```

Comment:
@jonlugoatx @reckenro This value is defined in the header file and it is used as the default value for the parameter Maximum_Time, which is a parameter in many APIs like FetchWaveform and ReadWaveform : 
![image](https://user-images.githubusercontent.com/47028785/117280257-0a7dde00-ae80-11eb-9f04-ac50239d6a7b.png)

1. If we add it as an enum, it might confuse the users as this enum will have only one value and they might think this is the only valid value. We should also add the enum tag to all the functions which have Maximum_Time as a parameter.
2. If we dont add this as an enum, the users will not be able to use it as NIDMM_VAL_TIME_LIMIT_AUTO and will have to pass the raw value itself.

There are many such single values defined for either attributes/function parameters like : 
- NIDMM_VAL_LATENCY_AUTO	
- NIDMM_VAL_SAMPLE_COUNT_INFINITE 
- NIDMM_VAL_SETTLE_TIME_AUTO

Should these be added or left out?

Reply:
I think this enum makes sense. It communicates the fact that when unspecified, the value ends up being zero and not -1 as the C function documentation indicates.

Reply:
I agree that an enum with just 1 value (+ the unspecified value) still makes sense. They should view parameters using enums the same regardless of the number of values in the enum. They can choose one of those pre-defined values present in the enum or pass in a raw value for the parameter.